compiler/rustc_session/src/options.rs

```diff
@@ -1401,6 +1401,8 @@ options! {
         "keep hygiene data after analysis (default: no)"),
     layout_seed: Option<u64> = (None, parse_opt_number, [TRACKED],
         "seed layout randomization"),
+    link_directives: bool = (true, parse_bool, [TRACKED],
+        "honour #[link] directives in crates (default: yes)"),
```

Comment:
```suggestion
        "honor #[link] directives in the compiled crate (default: yes)"),
```
Need to emphasize that it's about *this* crate and not all crates like `-Zlink_native_libraries`.

(Also American English.)

Reply:
OK, I pushed an update.